packages/circuit-ui/components/Card/Card.tsx

```diff
@@ -24,7 +24,10 @@ export interface CardProps extends HTMLProps<HTMLDivElement> {
    * The padding of the Card.
    */
   spacing?: 'mega' | 'giga';
-  ref: Ref<HTMLDivElement>;
+  /**
```

Comment:
Oh apparently the latest `next` was already extending the `HTMLDivElement` 😛 The only change is on this ref then

Reply:
Yeah, I fixed this recently (#1038) but forgot to make the `ref` optional 🤦🏻‍♂️